src/Common/Exception/CollectionIsEmpty.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the Geocoder package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license    MIT License
+ */
+
+namespace Geocoder\Exception;
+
+/**
+ * When you are trying to access an element on en empty collection.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
```

Comment:
Offtopic: https://www.databasesandlife.com/dont-use-the-author-javadoc-tag/ (I've noticed the original class had no `@author` tag: it proves the point that the tag is useless).